UTKcataloging/mods_to_rdf
# Add location to mapping decisions.

**JIRA Issue**: 

What Does this Do?
==================

This builds upon the initial location analysis PR and adds final decisions to the group mapping doc.

How Should This Be Tested?
==========================

Check to make sure that all tests pass. Check the content for consistency and justifiable property selections. Check for my many typos.

Additional Notes
================

Nothing changed significantly from analysis doc.
